tests/integrated/test-drift-instability/2fluid.cxx

```diff
@@ -3,415 +3,420 @@
+    Options* globalOptions = Options::getRoot();
+    Options* options = globalOptions->getSection("2fluid");
+    OPTION(options, AA, 2.0);
+    OPTION(options, ZZ, 1.0);
+
+    OPTION(options, estatic, false);
+    OPTION(options, ZeroElMass, false);
+    OPTION(options, zeff, 1.0);
+    OPTION(options, nu_perp, 0.0);
+    OPTION(options, ShearFactor, 1.0);
+
+    (globalOptions->getSection("Ni"))->get("evolve", evolve_ni, true);
+    (globalOptions->getSection("rho"))->get("evolve", evolve_rho, true);
+    (globalOptions->getSection("vi"))->get("evolve", evolve_vi, true);
+    (globalOptions->getSection("te"))->get("evolve", evolve_te, true);
+    (globalOptions->getSection("ti"))->get("evolve", evolve_ti, true);
+    (globalOptions->getSection("Ajpar"))->get("evolve", evolve_ajpar, true);
+
+    if (ZeroElMass)
+      evolve_ajpar = false; // Don't need ajpar - calculated from ohm's law
+
+    /*************** INITIALIZE LAPLACIAN SOLVERS ********/
+    phi_solver = Laplacian::create(globalOptions->getSection("phisolver"));
+    if (!estatic && !ZeroElMass) {
+      apar_solver = Laplacian::create(globalOptions->getSection("aparsolver"));
+    }
 
-  /************* SHIFTED RADIAL COORDINATES ************/
+    /************* SHIFTED RADIAL COORDINATES ************/
 
-  bool ShiftXderivs;
-  globalOptions->get("shiftXderivs", ShiftXderivs, false); // Read global flag
-  if(ShiftXderivs) {
-    ShearFactor = 0.0;  // I disappears from metric
-    b0xcv.z += I*b0xcv.x;
-  }
+    bool ShiftXderivs;
+    globalOptions->get("shiftXderivs", ShiftXderivs, false); // Read global flag
+    if (ShiftXderivs) {
+      ShearFactor = 0.0; // I disappears from metric
+      b0xcv.z += I * b0xcv.x;
+    }
 
-  /************** CALCULATE PARAMETERS *****************/
+    /************** CALCULATE PARAMETERS *****************/
 
-  rho_s = 1.02*sqrt(AA*Te_x)/ZZ/bmag;
-  fmei  = 1./1836.2/AA;
+    rho_s = 1.02 * sqrt(AA * Te_x) / ZZ / bmag;
+    fmei = 1. / 1836.2 / AA;
 
-  lambda_ei = 24.-log(sqrt(Ni_x)/Te_x);
-  lambda_ii = 23.-log(ZZ*ZZ*ZZ*sqrt(2.*Ni_x)/pow(Ti_x, 1.5));
-  wci       = 9.58e3*ZZ*bmag/AA;
-  nueix     = 2.91e-6*Ni_x*lambda_ei/pow(Te_x, 1.5);
-  nuiix     = 4.78e-8*pow(ZZ,4.)*Ni_x*lambda_ii/pow(Ti_x, 1.5)/sqrt(AA);
-  nu_hat    = zeff*nueix/wci;
+    lambda_ei = 24. - log(sqrt(Ni_x) / Te_x);
+    lambda_ii = 23. - log(ZZ * ZZ * ZZ * sqrt(2. * Ni_x) / pow(Ti_x, 1.5));
+    wci = 9.58e3 * ZZ * bmag / AA;
+    nueix = 2.91e-6 * Ni_x * lambda_ei / pow(Te_x, 1.5);
+    nuiix = 4.78e-8 * pow(ZZ, 4.) * Ni_x * lambda_ii / pow(Ti_x, 1.5) / sqrt(AA);
+    nu_hat = zeff * nueix / wci;
 
-  if(nu_perp < 1.e-10) {
-    mui_hat      = (3./10.)*nuiix/wci;
-  } else
-    mui_hat      = nu_perp;
+    if (nu_perp < 1.e-10) {
+      mui_hat = (3. / 10.) * nuiix / wci;
+    } else
+      mui_hat = nu_perp;
 
-  if(estatic) {
-    beta_p    = 1.e-29;
-  }else
-    beta_p    = 4.03e-11*Ni_x*Te_x/bmag/bmag;
+    if (estatic) {
+      beta_p = 1.e-29;
+    } else
```

Comment:
warning: statement should be inside braces [readability-braces-around-statements]

```cpp
} else
      ^
       {
```